xds/internal/balancer/outlierdetection/e2e_test/outlierdetection_test.go

```diff
@@ -367,3 +383,68 @@ func (s) TestNoopConfiguration(t *testing.T) {
 		t.Fatalf("error in expected round robin: %v", err)
 	}
 }
+
+// Test verifies that outlier detection doesn't eject subchannels created by
+// the new pickfirst balancer when pickfirst is a petiole policy itself. When
```

Comment:
I think that pickfirst is still not the petiole policy here. Based on this: https://github.com/grpc/proposal/blob/master/A61-IPv4-IPv6-dualstack-backends.md#use-pick_first-as-the-universal-leaf-policy, petiole policies are ones which delegate to pickfirst.

Reply:
Reworded to "non-leaf" policy and clarified that it means "not under a petiole policy".